deploy/docker/edumfa.py

```diff
@@ -20,14 +34,34 @@ def _getenv(key: str, default: str | None = None) -> str:
         return value
 
 
-SQLALCHEMY_DATABASE_URI = f"{_getenv('DB_DRIVER')}://{_getenv('DB_USER')}:{_getenv('DB_PASSWORD')}@{_getenv('DB_HOSTNAME')}/{_getenv('DB_DATABASE')}"
-SUPERUSER_REALM = _getenv("SUPERUSER_REALM", "super,administrators").split(",")
-SECRET_KEY = _getenv("SECRET_KEY")
-EDUMFA_PEPPER = _getenv("EDUMFA_PEPPER")
+def get_var(key: str, default: str | None = None) -> str:
+    """
+    Decide whether to get the variable from the environment or a file,
+    depending on if there is a environment variable with a _PATH-suffix
+    which overrides the environment variable without it.
+
+    :param key: The environment variable to get.
+    :param default: The value to return if the environment variable doesn't
+        exist.
+    :return: The variable's value or the default if given.
+    :raises ValueError: If the variable is not set and there is no default
+        value.
+    """
+    variable_with_path_value = getenv(f"{key}_PATH")
+    if variable_with_path_value:
+        return get_content_from_file(variable_with_path_value)
+    else:
+        return _getenv(key, default)
```

Comment:
```suggestion
def get_var(key: str, default: str | None = None) -> str:
    """
    Decide whether to get the variable from the environment or a file,
    depending on if there is a environment variable with a _FILE-suffix
    which overrides the environment variable without it.

    :param key: The environment variable to get.
    :param default: The value to return if the environment variable doesn't
        exist.
    :return: The variable's value or the default if given.
    :raises ValueError: If the variable is not set and there is no default
        value.
    """
    path = getenv(f"{key}_FILE")
    if path:
        return get_content_from_file(path)
    return _getenv(key, default)
```
The naming convention of secret files is `_FILE` not `_PATH`